components/feature/webnotifications/src/main/java/mozilla/components/feature/webnotifications/NativeNotificationBridge.kt

```diff
@@ -82,13 +80,14 @@ internal class NativeNotificationBridge(
     /**
      * Load an icon for a notification.
      */
-    private suspend fun loadIcon(url: Uri?, size: Size, isPrivate: Boolean): Bitmap? {
+    private suspend fun loadIcon(url: String?, iconUrl: String?, size: Size, isPrivate: Boolean): Bitmap? {
         url ?: return null
+        iconUrl ?: return null
```

Comment:
Should we have a test to verify how the caller of `loadIcon` reacts when  `iconUrl` is `null` and  we return `true`?

Reply:
Sure I'll add a test for this.  Thanks